Src/Model/Factory/FormatFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Aot\Model\Factory;
+
+class FormatFactory
+{
+    private $format = ['csv','json'];
+    private $path = 'Aot\Model\Adapter\\';
```

Comment:
`\` or `\\` ?

Reply:
if i put only one \ the ' is disable

Reply:
You mean `\M` and `\A` are escaped also? If you `\` as a string, always escape it

Reply:
i mean 'l\'ecole\' enseigne la vie' where stop the string?

Reply:
to escape a special character i use  \  . and  \  is a special character if he is use to escape a special character. and M and A are not special character then  \  is not a special character.

Reply:
http://php.net/language.types.string